tests/clang_parser.cpp

```diff
@@ -397,18 +402,35 @@ TEST(clang_parser, parse_fail)
 
 #include "btf_data.h"
 
-TEST(clang_parser, btf)
+static char* set_btf_test_env(void)
```

Comment:
Can we use a fixture class here? It'll help keep the setup/teardown code separate from the actual tests.

https://github.com/google/googletest/blob/master/googletest/docs/primer.md#test-fixtures-using-the-same-data-configuration-for-multiple-tests-same-data-multiple-tests